backend/src/apiserver/template/v2_template.go

```diff
@@ -256,7 +259,9 @@ func (t *V2Spec) RunWorkflow(modelRun *model.Run, options RunWorkflowOptions) (u
 		return nil, util.Wrap(err, "Failed to convert to PipelineJob RuntimeConfig")
 	}
 	job.RuntimeConfig = jobRuntimeConfig
-
+	if err = t.validatePipelineJobInputs(job); err != nil {
+		return nil, util.Wrap(err, fmt.Sprintf("invalid pipeline job: %s", err.Error()))
```

Comment:
nit: invalid pipeline job inputs

Reply:
Thank you! Updated.

Reply:
Missed this error message :) will add it.